src/client/setProtocol.js

```diff
@@ -38,7 +37,7 @@ module.exports = function (client, options) {
                 : client.profileKeys.signature
             }
           : null,
-        playerUUID: client.session?.selectedProfile?.id
+        playerUUID: !mcData.supportFeature('useChatSessions') ? client.session?.selectedProfile?.id : undefined
```

Comment:
Protodef will ignore fields if not applicable to version, version checks on them not needed

Reply:
I got errors without this. Might have been because of the MC data issues earlier